guess.py

```diff
@@ -5,18 +5,27 @@
 print("What is your name?")
 name = input()
 
-guess_count = 7
+guess_count = 4
 guess = 0
-print(name + " this is a guess game be sure to enter number that comes to mind.")
-while guess != number:
+print(name + ", this is a guess game be sure to enter number that comes to mind.")
+while guess < 4:
     guess = int(input('Enter any number:'))
-    if guess < number:
-        print("Wrong Guess")
-    elif guess > number:
-        print("Wrong Guess")
+    if guess % 2:
+        print("The answer is an odd number")
     else:
-        print("You got it right")
-
+        print("The answer is an even number")
+        break
+    if guess > number:
+        print("The answer is lower than the number")
+    else:
+        print("The answer is greater than the number")        
+        break
+    if guess == number:
+        print("Well done, you got it right")
+    else:
+        print("Wrong answer! you have run put attempts")
+    
```

Comment:
now, check which attempt theyre on.

```suggestion
    if num_of_attempts == 1:
          # this is where we give them the first hint.
          if guess % 2 == 0:
              print("The answer is an even number")
          else:
                print("the answer is an odd number")
       if num_of_attempts == 2:
            # give second hint here ...
```